packages/httpsnippet-client-api/test/tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "noImplicitAny": false,
-    "resolveJsonModule": true
+    "module": "Node16",
+    "resolveJsonModule": true,
+    "target": "ES2015"
```

Comment:
i don't really remember what the `target` field is for but this feels like it could be higher

Reply:
Bumped it to ES2022.